cmd/client/job-get.go

```diff
@@ -35,7 +35,7 @@ Metadata:
   Owner:	{{ .Metadata.Owner }}
   Trigger:	{{ .Metadata.Trigger }}
   Started:	{{ .Metadata.Created | toRFC3339 }}
-  Finished:	{{ .Metadata.Finished | toRFC3339 }}
+  Duration:	{{ .Metadata.Duration | toRFC3339 }}
```

Comment:
Oops, it looks like sometimes, `Finished` actually is the finished time and not a duration:

```
Metadata:
  Owner:           Jan Keromnes
  Trigger:         TRIGGER_MANUAL
  Started:         2020-02-21T10:19:58Z
  Finished:        2020-02-21T10:21:08Z
```

Reply:
Finished always is the finished time :)
It's just the UI that shows it as duration.